TimeServer/TimeServer/TimeServer/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Globalization;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.IO;
+
+namespace TimeServer;
+
+public static class Program {
+    static void Main(string[] arguments)
+    {
+        TcpListener? server = null;
+        try
+        {
+            server = new TcpListener(11);
+
+            server.Start();
+
+            for (;;)
+            {
+                Console.WriteLine("Connecting...");
+
+                TcpClient client = server.AcceptTcpClient();
+
+                Console.WriteLine("Connected!");
+
+                var response = DateTime.Now.ToString(CultureInfo.InvariantCulture);
+
+                StreamWriter streamWriter = new StreamWriter(client.GetStream());
```

Comment:
You were supposed to not use a StreamWriter here, yet, in order to learn about byte-buffers. But of course, this is the more elegant solution :)